### DISCUSSION TITLE
Auto reading in rest of row from primary key

### DISCUSSION THREAD

**Discussion by Crig001** [dbeaver/dbeaver#12671] [Q&A]
Okay so I am very new to mysql I have mysql server set up and have connected with dbeaver I have created two tables **employees** & **employee_training**, in **employees** there is a employee_id, then firstname, lastname etc... , and in **employee_training** there is a forigne key to employee_id is there a way for me to input say employee_id 1 and get the whole row of information from **employees** table into **employee_training**.

Any help would be great

Cheers

**Comment by LonwoLonwo** [comment]
Hello @Crig001 
If I understood you correctly,
you can use reference tool in Data viewer:

![2021-05-31 18_52_02-DBeaver Enterprise 21 1 0 - employee](https://user-images.githubusercontent.com/45152336/120218174-03a97780-c242-11eb-8b7a-7704d656be18.png)

or Reference panel in panel part:
![2021-05-31 18_52_32-DBeaver Enterprise 21 1 0 - 10149 (Не отвечает)](https://user-images.githubusercontent.com/45152336/120218224-1459ed80-c242-11eb-84e3-767538c0dbb3.png)

![2021-05-31 18_53_27-DBeaver Enterprise 21 1 0 - employee](https://user-images.githubusercontent.com/45152336/120218239-19b73800-c242-11eb-80a2-f52b3c096ff2.png)

**Reply by Crig001** [reply]
Hi thanks for you reply that is great and works however would you have to copy and paste the data or is there a way to automaticaly input the data into the corresponding rows ?

Cheers

**Reply by LonwoLonwo** [reply]
Nope, sorry. We don't have this tool.
Use "copy" + "paste"
